client/txn.go

```diff
@@ -280,6 +291,15 @@ func (txn *Txn) Commit(b *Batch) error {
 	return txn.Run(b)
 }
 
+// Rollback a transaction.
+func (txn *Txn) Rollback(b *Batch) error {
```

Comment:
I think the point of the current `Commit` method is efficiency, so duplicating its logic here is probably not really what we want. Maybe rename `Commit` to `EndInBatch` and give it a `commit bool` argument?